aplib.net-demo/Assets/Scripts/Entities/Enemy/RangedEnemy.cs

```diff
@@ -42,15 +53,30 @@ protected override void Update()
         Debug.DrawRay(transform.position, Vector3.up * 20, Color.green);
 
         // If the target is not directly visible but within vision range, move closer to the target.
-        if (!_rangedWeapon.EnemiesInLineOfSight() && _pathFind.GoalWithinRange(_visionRange)) _pathFind.SetStoppingDistance(1f);
+        if (!_rangedWeapon.EnemiesInLineOfSight() && _pathFind.GoalWithinRange(_visionRange))
+        {
+            _pathFind.SetStoppingDistance(1f);
```

Comment:
Should that 1f value be defined above so it can be set better as well?

Reply:
no, this is just an arbitrary small number to make the enemy move closer

Reply:
But it is used in multiple places, should that not be defined?